app.js

```diff
@@ -31,10 +34,14 @@ const givenName = document.getElementById('user-name');
 
 // Modal running
 submitNameBtn.addEventListener('click', () => {
-    appState.userName = inputName.value;
-    givenName.innerHTML = appState.userName;
-
-    modal.classList.add('invisible');
+    if(inputName.value === '') {
+        alert('You must enter your name there.');
```

Comment:
let's keep im mind, that we can handle it better by showing proper error, for example under the input, but it's not for this PR.